IdentityCore/src/broker_operation/request/MSIDBrokerOperationRequest.m

```diff
@@ -40,20 +40,11 @@ + (BOOL)fillRequest:(MSIDBrokerOperationRequest *)request
 keychainAccessGroup:(NSString *)keychainAccessGroup
      clientMetadata:(NSDictionary *)clientMetadata
             context:(id<MSIDRequestContext>)context
-              error:(NSError **)error
 {
     NSString *accessGroup = keychainAccessGroup ?: MSIDKeychainTokenCache.defaultKeychainGroup;
     __auto_type brokerKeyProvider = [[MSIDBrokerKeyProvider alloc] initWithGroup:accessGroup];
-    NSError *brokerError = nil;
     NSString *base64UrlKey = [brokerKeyProvider base64BrokerKeyWithContext:context
```

Comment:
nit: should we differentiate a case when we failed to create the broker key from the case when it wasn't expected to be created? (e.g. if we always fail to create broker key in MSAL case, I'd prefer that to be a failure)

Reply:
I think it is better to log it (what we currently do). As long as it is optional on protocol level, I think we should be consistent with other apps which will use our protocol.